Tests/Reqnroll.VisualStudio.Specs/Reqnroll.VisualStudio.Specs.csproj

```diff
@@ -6,6 +6,9 @@
     <LangVersion>latest</LangVersion>
     <NoWarn>VSTHRD200<!--Use "Async" suffix in names of methods that return an awaitable type.--></NoWarn>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
All changes in this project file should be undone, these are done by visual studio by mistake. They are not needed.

Reply:
Removed